### DISCUSSION TITLE
3.3.6

### DISCUSSION THREAD

**Discussion by mathiascode** [nicotine-plus/nicotine-plus#3159] [Releases]
WINDOWS USERS: The installer format has changed in Nicotine+ 3.2.0. If you are upgrading from Nicotine+ 3.1.1 or earlier, please uninstall Nicotine+ first (this will not remove your existing settings).

### Changes

 * Hide picture view when no user profile picture is present
 * GTK 4: Stop using old GL renderer
 * Windows: Reduced package size by switching to MSYS2's Clang environment

### Corrections

 * Important: Don't delete other unrelated files from incomplete download folder
 * Fixed a crash when editing and removing list rows in some cases
 * Fixed a regression where arrow keys no longer selected items in comboboxes
 * Minor fixes related to the tray icon implementation
 * GTK 3: Fixed an issue where the chat text entry could disappear
 * macOS: Fixed an issue where opening files and links no longer worked
 * macOS: Fixed a few keyboard shortcuts that no longer worked

### Issues closed on GitHub

 * 'Open in File Manager' does not work on 3.3.5 in macOS Monterey ([#3143](https://github.com/nicotine-plus/nicotine-plus/issues/3143))
 * Issues with selecting text on 3.3.5 in macOS Monterey ([#3144](https://github.com/nicotine-plus/nicotine-plus/issues/3144))
 * Horizontal adjustment of user profile ([#2865](https://github.com/nicotine-plus/nicotine-plus/issues/2865))
 * Files deleted on quit ([#3152](https://github.com/nicotine-plus/nicotine-plus/issues/3152))
 * the window does not close automatically and gets in the way ([#3157](https://github.com/nicotine-plus/nicotine-plus/issues/3157))

<hr /><em>This discussion was created from the release <a href='https://github.com/nicotine-plus/nicotine-plus/releases/tag/3.3.6'>3.3.6</a>.</em>

**Comment by illimay** [comment]
i'm getting an error... 
cannot connect to server slsknet.org error 10060 unknown error
it keeps retrying but never connects :/

**Reply by slook** [reply]
Have you set a Network Interface binding to a specific adapter in Preferences >Network > Miscellaneous or is it blank?

**Reply by illimay** [reply]
it's left blank. but it works when i use a vpn. i don't understand... is my isp doing something?

**Reply by slook** [reply]
Are you able to visit the https://www.slsknet.org website?

**Reply by illimay** [reply]
yes. i'm able to access the website without a vpn.

**Reply by slook** [reply]
Then it doesn't seem as if the server maintainer has banned your IP address, otherwise your username would probably be banned aswell. It doesn't appear to be any problem in Nicotine+, since you are able to connect using an alternative route.

It's perhaps more likely that your ISP is disallowing the particular protocol port or traffic pattern, because to them slsk packets probably look like some type of spam botnet.

**Reply by illimay** [reply]
understood. thank you.
